puppet/zulip/files/cron.d/calculate-first-visible-message-id

```diff
@@ -0,0 +1,6 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+0 7 * * * zulip /home/zulip/deployments/current/manage.py calculate_first_visible_message_id --lookback-period 24
```

Comment:
Let's name this flag `--lookback-hours` -- that way when looking at an invocation like this it's transparent what it means.

Reply:
Also, like your usage message recommends, this should be a bit more than 24 hours :) -- let's make it 30.